package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@yfi/yearn-web-lib",
+  "version": "0.1.0",
+  "description": "Basic web library for Yearn's projects",
+  "license": "MIT",
```

Comment:
Let's remove that for now 🙏

Reply:
The license? I actually didn't add it initially, but then there was this warning about the license missing that was annoying, happy to remove. Removed with https://github.com/Major-Eth/yearn-web-lib/pull/8/commits/4cad7725edb01f0cd3a7d9020e8f444ad4d02cc4

```cmd
➜  yearn-web-lib git:(issue-1) ✗ yarn
yarn install v1.22.17
warning package.json: No license field
warning @yearn/yearn-web-lib@0.1.0: No license field
```